eng/Versions.props

```diff
@@ -33,16 +33,16 @@
     <ILDAsmPackageVersion>6.0.0-rtm.21518.12</ILDAsmPackageVersion>
     <MicrosoftVisualStudioLanguageServerClientPackagesVersion>17.3.3101</MicrosoftVisualStudioLanguageServerClientPackagesVersion>
     <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>17.5.20-preview</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
-    <MicrosoftVisualStudioShellPackagesVersion>17.5.0-preview-2-33117-317</MicrosoftVisualStudioShellPackagesVersion>
+    <MicrosoftVisualStudioShellPackagesVersion>17.6.35295-preview.2</MicrosoftVisualStudioShellPackagesVersion>
```

Comment:
This _might_ break our integration tests if this is moving to a version newer than what VS is installed in our integration test machines. Something to be on the lookout for.

Reply:
The ServiceHub.Framework update that I'm going for here _is_ new for 17.6 Preview **2**. If your integration tests don't run on preview 2, I suppose it's premature to upgrade this dependency in the first place. When do you think you'll run with Preview 2? After it is released publicly?

Reply:
Yes, a bit after the public release actually. Right now the latest version we could be using is still 17.5p6
https://dev.azure.com/dnceng/internal/_wiki/wikis/DNCEng%20Services%20Wiki/899/VS2022-Upgrade-Schedule